config/tests/samples/create/harness.go

```diff
@@ -145,6 +151,9 @@ func NewHarnessWithOptions(ctx context.Context, t *testing.T, opts *HarnessOptio
 	// creating multiple managers for tests will fail if more than one
 	// manager tries to bind to the same port.
 	kccConfig.ManagerOptions.HealthProbeBindAddress = "0"
+	// Enable the configuration of state-into-spec default value in kccmanager
+	kccConfig.StateIntoSpecDefaultValue = k8s.StateIntoSpecDefaultValueV1Beta1
```

Comment:
This was added in a previous [PR](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1152) but got removed later(for unknown reasons). I think we want to add it back.